src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    public static void goodbye()
+    {
```

Comment:
Do remember to follow Egyptian style for the curly braces. :)

Reply:
Thank you! It will be solved in the next commit.